internal/pkg/print/markdown/markdown.go

```diff
@@ -27,6 +27,9 @@ func Print(document *doc.Doc, settings settings.Settings) (string, error) {
 	}
 
 	if document.HasOutputs() {
+		if document.HasInputs() {
```

Comment:
Please move this block down between the sorting and the call to `printOutputs`.

Reply:
Done.